.circleci/config.yml

```diff
@@ -337,7 +331,6 @@ workflows:
               exp: ['A', 'B', 'C']
           <<: *push_and_pr_builds
           requires:
-            - 'Compute Merge Commit'
```

Comment:
Also remove this from the commented out `Performance Tests`.

Reply:
Done